### PR TITLE
Rename deptId to departmentId

### PR DESCRIPTION
**Pull request by zkhin**
Fixes: https://github.com/SeedCompany/cord-api-v3/issues/1144

### CODE REVIEW

File: src/components/project/dto/project.dto.ts [hunk 1]
```diff
@@ -41,7 +41,7 @@ class Project extends Resource {
     description: 'The legacy department ID',
     deprecationReason: 'Use `Project.departmentId` instead',
```
Comment (CarsonF) [inline src/components/project/dto/project.dto.ts:42]: This needs to be moved to the field resolver

File: src/components/project/project.resolver.ts [hunk 2]
```diff
@@ -74,7 +74,15 @@ export class ProjectResolver {
     description: 'The legacy department ID',
   })
   departmentId(@Parent() project: Project): SecuredString {
-    return project.deptId;
+    return project.departmentId;
+  }
```
Comment (CarsonF) [inline src/components/project/project.resolver.ts:78]: This one isn't needed anymore

**Review by CarsonF — CHANGES_REQUESTED** [review]

File: src/components/project/dto/project.dto.ts [hunk 3]
```diff
@@ -39,9 +39,9 @@ class Project extends Resource {
 
   @Field({
     description: 'The legacy department ID',
-    deprecationReason: 'Use `Project.departmentId` instead',
+    deprecationReason: 'Moved to the field resolver',
```
Comment (CarsonF) [inline src/components/project/dto/project.dto.ts:42]: Sorry for the confusion, I mean't the `deprecatedReason` should be removed here (`departmentId` is not deprecated) and added to the ResolveField's options for `deptId` which, still is, deprecated.

**Review by CarsonF — CHANGES_REQUESTED** [review]

File: src/components/project/project.resolver.ts [hunk 4]
```diff
@@ -70,11 +70,12 @@ export class ProjectResolver {
       : undefined;
   }
 
+  /** @deprecated Moved from field definition in DTO to here */
   @ResolveField(() => SecuredString, {
     description: 'The legacy department ID',
   })
```
Comment (CarsonF) [inline src/components/project/project.resolver.ts:76]:
```suggestion
  @ResolveField(() => SecuredString, {
    description: 'The legacy department ID',
    deprecationReason: 'Use `Project.departmentId` instead',
  })
```

Comment (zkhin) [reply src/components/project/project.resolver.ts:76]: Thanks.

**Review by CarsonF — APPROVED** [review]
Thanks @zkhin!

**Pull request merged**